postgres_copy/copy_from.py

```diff
@@ -101,6 +101,8 @@ def save(self, silent=False, stream=sys.stdout):
                 "%s records loaded\n" % intcomma(insert_count)
             )
 
+        return insert_count
```

Comment:
I'd like a unit test yet that verifies this number is returned, and that it is accurate.